homeassistant/components/device_tracker/asuswrt.py

```diff
@@ -283,15 +283,16 @@ def run_command(self, command):
             lines = self._ssh.before.split(b'\n')[1:-1]
             return [line.decode('utf-8') for line in lines]
         except exceptions.EOF as err:
-            _LOGGER.error("Connection refused. SSH enabled?")
+            _LOGGER.error("Connection refused. SSH enabled? Check known_hosts "
+                          "under the homeassistant user home directory.")
```

Comment:
Unfortunately we cannot simply print the exception: see [here](https://github.com/home-assistant/home-assistant/issues/2978#issuecomment-244557798)